experiment/build/generate_cloudbuild.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Generates Cloud Build specification"""
+
+import argparse
+import json
+import yaml
+
+from experiment.build import docker_images
+
+
+# TODO: Add unit test for this.
+def create_cloud_build_spec(buildable_images, docker_registry):
+    """Returns Cloud Build specificatiion."""
+
+    cloud_build_spec = {}
+    cloud_build_spec['steps'] = []
+    cloud_build_spec['images'] = []
+
+    for name, image in buildable_images.items():
+        step = {}
+        step['id'] = name
+        step['name'] = 'gcr.io/cloud-builders/docker'
+        step['args'] = []
+        step['args'] += ['--tag', image['tag']]
```

Comment:
I think we need two tags here as the YAML files have.
This is so that (for example) base-image can be pushed to gcr.io/fuzzbench-test/base-image and base-builder can inherit from gcr.io/fuzzbench/base-image instead of dealing with the mess of using arguments to decide what to inherit from.

Reply:
:-( I suspect this trick won't work with buildkit since I assume buildkit will be "smart" by pulling from gcr.io/fuzzbench/base-image when it should really pull from gcr.io/fuzzbench-test/base-image if the repo we are using is gcr.io/fuzzbench-test.

Reply:
Afaik, we can tell buildkit to `--cache-from` any docker registry/image.
(I'm not sure though why'd we need a separate gcr.io/fuzzbench and gcr.io/fuzzbench-test registy). 
Anyhow, if we need that i'd have a separate `--push-docker-registry` and `--cache-docker-registry` flag.